oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -361,7 +366,23 @@ private HttpResponse getMetadataResponse(
     request.setThrowExceptionOnExecuteError(false);
     HttpResponse response;
     try {
+      String requestMessage;
+      String responseMessage;
+      if (requestType.equals(RequestType.ID_TOKEN_REQUEST)) {
+        requestMessage = "Auth get metadata sending request for id token";
+        responseMessage = "Auth get metadata received response for id token";
+      } else if (requestType.equals(RequestType.ACCESS_TOKEN_REQUEST)) {
+        requestMessage = "Auth get metadata sending request for access token";
+        responseMessage = "Auth get metadata received response for access token";
+      } else {
+        // TODO: this includes get universe domain and get default sa.
+        // refactor for more clear logging message.
+        requestMessage = "Auth get metadata sending request";
+        responseMessage = "Auth get metadata received response";
+      }
```

Comment:
nit: How about `Sending request for ...` and `Received response for ...` without the `Auth get metadata` prefix? 

I don't think we need the specify it's Auth (unless that's required) and `get metadata` seems to be a implementation (i.e. private method) that is calling it

Reply:
At least I believe the logs will show up as something like `INFO  com.google.auth.ComputeEngineCredentials -- Sending request for ID Token` or something similar